Source/Core/Core/HW/EXI_DeviceEthernet.cpp

```diff
@@ -358,6 +358,12 @@ void CEXIETHERNET::MXCommandHandler(u32 data, u32 size)
 		data &= (data & 0xff) ^ 0xff;
 		goto write_to_register;
 
+	case BBA_TXFIFOCNT:
```

Comment:
This probably works fine, but I think it would be better to do the address check in the write_to_register loop. That way, even things starting below BBA_TXFIFOCNT and overwriting it would be caught.